src/directives/toggle/toggle.directive.ts

```diff
@@ -65,21 +71,29 @@ export class IgxToggleDirective implements IToggleView, OnInit, OnDestroy {
         if (!this.collapsed) { return; }
 
         const player = this.animationActivation();
-        player.onStart(() => this.collapsed = !this.collapsed);
+        player.onStart(() => {
+            // this.collapsed = !this.collapsed;
```

Comment:
If we'll leave it commented we need to remove onStart at all

Reply:
Done